### DISCUSSION TITLE
Make apks available on this repository's Github releases

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#12739] [Ideas]
https://osmand.net/downloads is nice and all, but by making the release apks available through Github releases as well, we could simply follow the repo and be notified when new releases are added, for updates and such. I'm not interested in play store or f-droid, and I'm probably not the only one.

**Comment by pebogufi** [comment]
Did you have a look at 
https://download.osmand.net/releases/
or do you mean something different?
